### DISCUSSION TITLE
No geoip data with latest version

### DISCUSSION THREAD

**Discussion by vitobotta** [plausible/analytics#1306] [Self-Hosted Support] [answered]
Hi! I installed Plausible in a new Kubernetes cluster with the latest Docker image. Everything works fine apart from the geolocation because no data about the countries is displayed on the dashboard. The release notes say that it is no longer necessary to add the geoip container. I have a Maxmind license, how do I configure Plausible so that it can download the Maxmind database and use that? Thanks!

**Comment by nakoo** [comment, marked as answer]
Have you checked their docs? I’m currently running the latest version, but it doesn’t have that issue.

[https://plausible.io/docs/self-hosting#1-maxmind-geolocation-database](https://plausible.io/docs/self-hosting#1-maxmind-geolocation-database)

**Reply by vitobotta** [reply]
> Have you checked their docs? I’m currently running the latest version, but it doesn’t have that issue.
> 
> https://plausible.io/docs/self-hosting#1-maxmind-geolocation-database

I have, but the release notes for the latest versions say that a separate container for the geoip stuff is no longer needed, so that page seems out of date? I will try that way again anyway, thanks.

**Comment by vitobotta** [comment]
No change. Even with the separate container I see that it downloads the database from Maxmind but it's not in use it seems

**Reply by nakoo** [reply]
Have you added `X-Forwarded-For` header to your backend side?

**Reply by vitobotta** [reply]
Yes. I'm in Kubernetes and the ingress controller is correctly configured to pass the client IP. I use this in various other apps.

**Comment by vitobotta** [comment]
Any suggestions on what else I could try? Thanks!
